cli/compose/loader/loader_test.go

```diff
@@ -1324,16 +1326,23 @@ services:
     image: busybox
     extra_hosts:
       - "zulu:162.242.195.82"
+      - "whiskey=162.242.195.83"
       - "alpha:50.31.209.229"
       - "zulu:ff02::1"
-      - "host.docker.internal:host-gateway"
+      - "whiskey=ff02::2"
```

Comment:
is having two hosts named "whiskey" (one ipv4 and one ipv6) intentional for this test?

Reply:
Yes, it's the same as the two existing "zulu" entries. In an '/etc/hosts' file, hosts get separate lines for IPv4/IPv6 addresses. (And, more whiskey is always good.)